delocate/delocating.py

```diff
@@ -704,6 +830,16 @@ def delocate_wheel(
             install_id_prefix=DLC_PREFIX + relpath(lib_sdir, wheel_dir),
         )
         rewrite_record(wheel_dir)
+        if check_wheel_name or fix_wheel_name:
+            wheel_name = os.path.basename(in_wheel)
+            new_name = _update_wheel_name(wheel_name, wheel_dir)
+            if check_wheel_name and new_name != wheel_name:
+                raise DelocationError(
+                    "Wheel name does not satisfy minimal package requirements"
+                )
```

Comment:
You mean to provide a list of files that have higher system required or only provide new and old tag? 

If first, should I return new name and list of problematic files or perform scan again to discover (it will require more code, but the function to determine wheel name will be more readable)?

Reply:
You're right, that could make it more complicated.

I was mostly worrying about people posting bug reports with this error message being the only context given. If there was a simple response to that kind of post then I'd like it to be included in the error message itself.

I was only thinking of the 2nd option since this code should already have that much information, but your 1st option would be much more helpful for anyone encountering this error. I'll accept either.